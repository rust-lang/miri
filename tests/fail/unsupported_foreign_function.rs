fn main() {
    extern "Rust" {
        fn foo();
    }

    unsafe {
        foo(); //~ ERROR: unsupported operation: can't call foreign function: foo
    }
}
