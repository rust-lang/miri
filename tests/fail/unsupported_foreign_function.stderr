error: unsupported operation: can't call foreign function: foo
  --> $DIR/unsupported_foreign_function.rs:LL:CC
   |
LL |         foo();
   |         ^^^^^ can't call foreign function: foo
   |
   = help: this is likely not a bug in the program; it indicates that the program performed an operation that the interpreter does not support
   = note: BACKTRACE:
   = note: inside `main` at $DIR/unsupported_foreign_function.rs:LL:CC

note: some details are omitted, run with `MIRIFLAGS=-Zmiri-backtrace=full` for a verbose backtrace

error: aborting due to previous error

